package.json

```diff
@@ -28,25 +30,40 @@
     {
       "name": "Christopher Thorn",
       "email": "chris.thorn@lytics.io"
+    },
+    {
+      "name": "Lauren Yeiser",
+      "email": "lauren.yeiser@lytics.io"
     }
-  ],
+    ],
+  "author": "",
   "license": "MIT",
-  "dependencies": {
-    "ember-cli-htmlbars": "0.7.4"
-  },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.0",
-    "ember-cli": "0.2.0",
-    "ember-cli-dependency-checker": "0.0.7",
-    "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-qunit": "^0.3.9",
-    "express": "^4.8.5",
-    "glob": "^4.0.5"
+    "broccoli-asset-rev": "^2.2.0",
+    "ember-cli": "1.13.13",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-content-security-policy": "0.4.0",
+    "ember-cli-dependency-checker": "^1.1.0",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
+    "ember-cli-ic-ajax": "0.2.4",
+    "ember-cli-inject-live-reload": "^1.3.1",
+    "ember-cli-qunit": "^1.0.4",
+    "ember-cli-release": "0.2.8",
+    "ember-cli-sri": "^1.2.0",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-data": "1.13.15",
+    "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-export-application-global": "^1.0.4",
+    "ember-disable-prototype-extensions": "^1.0.0",
+    "ember-try": "~0.0.8"
   },
-  "description": "This is a collection of generic components used by Lytics Inc.",
```

Comment:
The description also got moved and updated. Sorry I didn't catch this on my first pass through.